packages/fts/src/parser.ts

```diff
@@ -250,6 +250,11 @@ function addParamsDeclaration(
     const name = param.getName()
     const structure = param.getStructure()
 
+    if (structure.isRestParameter) {
+      builder.definition.params.schema = { additionalProperties: true };
+      continue;
```

Comment:
`prettier` is enabled on this codebase and this code doesn't conform to the settings. i'm surprised the pre-commit `lint-staged` didn't catch this, but let's make sure to run prettier before merging.

Reply:
Odd, not sure why my prettier didn't pick up your config 🤔

Reply:
Addressed